packages/sdk-codegen-scripts/src/legacyGenerator.ts

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { log } from '@looker/sdk-codegen-utils'
+import { IGeneratorSpec, legacyLanguages } from '@looker/sdk-codegen'
+import { ISDKConfigProps } from './sdkConfig'
+import { run } from './nodeUtils'
+import { fetchLookerVersions, logConvertSpec } from './fetchSpec'
+
+/**
+ * Returns the last version in the .ini api_versions comma-delimited list
+ * @param {ISDKConfigProps} props
+ * @returns {string}
+ */
+const defaultApiVersion = (props: ISDKConfigProps) => {
+  const versions = (props.api_versions || '4.0').split(',')
+  return versions[versions.length - 1]
+}
+
+/**
+ * Generate API bindings with the OpenAPI legacy code generator
+ * DEPRECATED: This is using the legacy code generator perform the generation for specific API version, configuration,
+ * and language
+ *
+ * @param {string} fileName specification file name
+ * @param {IGeneratorSpec} spec for language generator options
+ * @param {ISDKConfigProps} props SDK configuration properties
+ * @returns {Promise<Buffer | string>}
```

Comment:
I think IntelliJ is generating old style doc comments for you

Reply:
That's just code I moved from legacy.ts to legacyGenerator.ts. Comment cleanup is a batch task I'm hoping to automate, possibly with a lint fix.